backend/app/Models/BookMark.php

```
@@ -0,0 +1,136 @@
+     */
+    public function countIncrease(): void
+    {
+        static::withoutTimestamps(fn () => $this->increment('count'));
+    }
+
+    /**
+     * データベース上のデータが、自分が取得した後に更新されていないか確認する。
+     */
+    public function hasBeenUpdatedSinceRetrieval(string $fetchedAt): bool
+    {
+        return $this->updated_at?->gt(Carbon::parse($fetchedAt)) ?? false;
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Carbon' in method 'hasBeenUpdatedSinceRetrieval'. (undefined)

(StaticAccess)